assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -11,9 +11,12 @@ xformers==0.0.20
 mlflow==2.3.1
 mlflow-skinny==2.3.1
 azure-ai-contentsafety==1.0.0b1
-azure-ai-mlmonitoring==0.1.0a3
 azure-identity==1.13.0
 azure-mgmt-cognitiveservices==13.4.0
 azureml-inference-server-http==0.8.4
 azureml-core==1.52.0
-azureml-mlflow==1.52.0
\ No newline at end of file
+azureml-mlflow==1.52.0
+cryptography==41.0.2
```

Comment:
you may want to bump this to 41.0.3. It's due by September 1st though, so you can do that separately